specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/integrationRuntime.json

```diff
@@ -0,0 +1,2837 @@
+          },
+          {
+            "name": "integrationRuntime",
+            "description": "Integration runtime resource definition.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/IntegrationRuntimeResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/IntegrationRuntimeResource"
+            }
+          },
+          "default": {
+            "description": "An error response received from PUT integration runtime operation.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "Create integration runtime": {
+            "$ref": "./examples/IntegrationRuntimes_Create.json"
+          }
+        }
+      },
+      "delete": {
+        "operationId": "IntegrationRuntimes_Delete",
+        "summary": "Delete integration runtime",
+        "description": "Delete an integration runtime",
+        "tags": [
+          "IntegrationRuntime"
+        ],
+        "deprecated": false,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./workspace.json#/parameters/workspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/IntegrationRuntimeNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK."
+          },
+          "204": {
+            "description": "No Content."
+          },
+          "default": {
+            "description": "An error response received from DELETE integration runtime operation.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "Delete integration runtime": {
+            "$ref": "./examples/IntegrationRuntimes_Delete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/integrationRuntimes/{integrationRuntimeName}/nodes/{nodeName}/ipAddress": {
```

Comment:
Please use a verb here - e.g `getIpAddress` rather than just `ipAddress`

Reply:
> Please use a verb here - e.g `getIpAddress` rather than just `ipAddress`

Hi anothony, change api needs backend team support. The backend just use `ipAddress`.
And I don't think changing it to `getIpAddress`  makes sense. Usually we use different http method type to stands for the meaning for example put method means creating ipAddress.

Reply:
> Please use a verb here - e.g `getIpAddress` rather than just `ipAddress`

Hi @anthony-c-martin I understand your point and I agree with you. But the datafactory is an existing service, synapse reused their some functionality, backend team won't change the api path because the effort is large. Can we ignore this?